doc/source/whatsnew/v0.20.0.txt

```diff
@@ -251,3 +251,4 @@ Bug Fixes
 
 
 - Require at least 0.23 version of cython to avoid problems with character encodings (:issue:`14699`)
+- Bug in ``lib.maybe_convert_objects()`` in which unsigned 64-bit integers were not being properly converted (:issue:`4471`)
```

Comment:
this is not public. I would refer to this more generically (maybe also construct a test which actually hits this in practice from a user perspective)